Can't build the latest beta UR Robot Driver

# Versions
 - ROS Driver version: 0.0.3, Melodic

# Problem

Hello 

I have a problem with building the latest ur_robot_driver ROS package. We got yesterday our new UR10e and after I noticed that you are restructured the new cartesian repositories in github, I decided to build my UR packages completely from scratch. So I deleted all UR packages and in my catkin folder the devel and build folders. I cloned the following gits:

git clone https://github.com/UniversalRobots/Universal_Robots_ROS_controllers_cartesian.git
git clone https://github.com/UniversalRobots/Universal_Robots_ROS_passthrough_controllers.git
git clone -b beta-testing https://github.com/UniversalRobots/Universal_Robots_ROS_Driver.git
git clone -b beta-testing-boost https://github.com/UniversalRobots/Universal_Robots_Client_Library.git
git clone https://github.com/UniversalRobots/Universal_Robots_ROS_cartesian_control_msgs.git
git clone https://github.com/ros-industrial/ur_msgs.git
git clone https://github.com/UniversalRobots/Universal_Robots_ROS_scaled_controllers.git
git clone https://github.com/fzi-forschungszentrum-informatik/cartesian_controllers.git
git clone https://github.com/fmauch/universal_robot

And when I try to build the ur_robot_driver I got the following error (see end of this post). It's quite confusing which packages we should now use for the latest beta driver. Because the main page here recommends [this](https://github.com/fzi-forschungszentrum-informatik/cartesian_ros_control/tree/beta-testing) and this contains for example the cartesian_ros_control package which [this](https://github.com/UniversalRobots/Universal_Robots_ROS_controllers_cartesian) repo doesn't contain.

I hope someone can help me or did I mixed up some of the packages?

Thanks in advance

> Errors     << ur_robot_driver:make /home/lg/catkin_ws/logs/ur_robot_driver/build.make.000.log                                                                                
In file included from /home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:28:0:
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/include/ur_robot_driver/hardware_interface.h:244:3: error: ‘cartesian_ros_control’ does not name a type
   cartesian_ros_control::CartesianStateInterface cart_interface_;
   ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/include/ur_robot_driver/hardware_interface.h:245:3: error: ‘cartesian_ros_control’ does not name a type
   cartesian_ros_control::TwistCommandInterface twist_interface_;
   ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/include/ur_robot_driver/hardware_interface.h:246:3: error: ‘cartesian_ros_control’ does not name a type
   cartesian_ros_control::PoseCommandInterface pose_interface_;
   ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/include/ur_robot_driver/hardware_interface.h: In constructor ‘ur_driver::HardwareInterface::HardwareInterface()’:
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/include/ur_robot_driver/hardware_interface.h:282:20: warning: ‘ur_driver::HardwareInterface::cartesian_pose_command_’ will be initialized after [-Wreorder]
   urcl::vector6d_t cartesian_pose_command_;
                    ^~~~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/include/ur_robot_driver/hardware_interface.h:271:20: warning:   ‘urcl::vector6d_t ur_driver::HardwareInterface::joint_positions_’ [-Wreorder]
   urcl::vector6d_t joint_positions_;
                    ^~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:56:1: warning:   when initialized here [-Wreorder]
 HardwareInterface::HardwareInterface()
 ^~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp: In member function ‘virtual bool ur_driver::HardwareInterface::init(ros::NodeHandle&, ros::NodeHandle&)’:
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:336:23: error: ‘JointTrajectoryHandle’ is not a member of ‘hardware_interface’
   hardware_interface::JointTrajectoryHandle joint_trajectory_handle = hardware_interface::JointTrajectoryHandle(
                       ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:336:23: note: suggested alternative: ‘JointTrajectory’
   hardware_interface::JointTrajectoryHandle joint_trajectory_handle = hardware_interface::JointTrajectoryHandle(
                       ^~~~~~~~~~~~~~~~~~~~~
                       JointTrajectory
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:340:23: error: ‘using JointTrajectoryInterface = class hardware_interface::TrajectoryInterface<control_msgs::FollowJointTrajectoryGoal_<std::allocator<void> >, control_msgs::FollowJointTrajectoryFeedback_<std::allocator<void> > > {aka class hardware_interface::TrajectoryInterface<control_msgs::FollowJointTrajectoryGoal_<std::allocator<void> >, control_msgs::FollowJointTrajectoryFeedback_<std::allocator<void> > >}’ has no member named ‘registerHandle’; did you mean ‘registerGoalCallback’?
   jnt_traj_interface_.registerHandle(joint_trajectory_handle);
                       ^~~~~~~~~~~~~~
                       registerGoalCallback
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:340:38: error: ‘joint_trajectory_handle’ was not declared in this scope
   jnt_traj_interface_.registerHandle(joint_trajectory_handle);
                                      ^~~~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:343:23: error: ‘CartesianTrajectoryHandle’ is not a member of ‘hardware_interface’
   hardware_interface::CartesianTrajectoryHandle cartesian_trajectory_handle =
                       ^~~~~~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:343:23: note: suggested alternative: ‘CartesianTrajectory’
   hardware_interface::CartesianTrajectoryHandle cartesian_trajectory_handle =
                       ^~~~~~~~~~~~~~~~~~~~~~~~~
                       CartesianTrajectory
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:348:24: error: ‘using CartesianTrajectoryInterface = class hardware_interface::TrajectoryInterface<cartesian_control_msgs::FollowCartesianTrajectoryGoal_<std::allocator<void> >, cartesian_control_msgs::FollowCartesianTrajectoryFeedback_<std::allocator<void> > > {aka class hardware_interface::TrajectoryInterface<cartesian_control_msgs::FollowCartesianTrajectoryGoal_<std::allocator<void> >, cartesian_control_msgs::FollowCartesianTrajectoryFeedback_<std::allocator<void> > >}’ has no member named ‘registerHandle’; did you mean ‘registerGoalCallback’?
   cart_traj_interface_.registerHandle(cartesian_trajectory_handle);
                        ^~~~~~~~~~~~~~
                        registerGoalCallback
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:348:39: error: ‘cartesian_trajectory_handle’ was not declared in this scope
   cart_traj_interface_.registerHandle(cartesian_trajectory_handle);
                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:348:39: note: suggested alternative: ‘cartesian_pose_command_’
   cart_traj_interface_.registerHandle(cartesian_trajectory_handle);
                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~
                                       cartesian_pose_command_
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:350:3: error: ‘cartesian_ros_control’ has not been declared
   cartesian_ros_control::CartesianStateHandle cart_state_handle("base", "tool0_controller", &cart_pose_, &cart_twist_,
   ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:352:3: error: ‘cart_interface_’ was not declared in this scope
   cart_interface_.registerHandle(cart_state_handle);
   ^~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:352:3: note: suggested alternative: ‘js_interface_’
   cart_interface_.registerHandle(cart_state_handle);
   ^~~~~~~~~~~~~~~
   js_interface_
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:352:34: error: ‘cart_state_handle’ was not declared in this scope
   cart_interface_.registerHandle(cart_state_handle);
                                  ^~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:353:3: error: ‘twist_interface_’ was not declared in this scope
   twist_interface_.registerHandle(
   ^~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:353:3: note: suggested alternative: ‘js_interface_’
   twist_interface_.registerHandle(
   ^~~~~~~~~~~~~~~~
   js_interface_
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:354:7: error: ‘cartesian_ros_control’ has not been declared
       cartesian_ros_control::TwistCommandHandle(cart_interface_.getHandle("tool0_controller"), &twist_command_));
       ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:357:3: error: ‘pose_interface_’ was not declared in this scope
   pose_interface_.registerHandle(
   ^~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:357:3: note: suggested alternative: ‘js_interface_’
   pose_interface_.registerHandle(
   ^~~~~~~~~~~~~~~
   js_interface_
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:358:7: error: ‘cartesian_ros_control’ has not been declared
       cartesian_ros_control::PoseCommandHandle(cart_interface_.getHandle("tool0_controller"), &pose_command_));
       ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp: In member function ‘virtual void ur_driver::HardwareInterface::read(const ros::Time&, const ros::Duration&)’:
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:562:27: error: ‘using JointTrajectoryInterface = class hardware_interface::TrajectoryInterface<control_msgs::FollowJointTrajectoryGoal_<std::allocator<void> >, control_msgs::FollowJointTrajectoryFeedback_<std::allocator<void> > > {aka class hardware_interface::TrajectoryInterface<control_msgs::FollowJointTrajectoryGoal_<std::allocator<void> >, control_msgs::FollowJointTrajectoryFeedback_<std::allocator<void> > >}’ has no member named ‘getHandle’
       jnt_traj_interface_.getHandle("joint_trajectory_handle").setFeedback(feedback);
                           ^~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:604:28: error: ‘using CartesianTrajectoryInterface = class hardware_interface::TrajectoryInterface<cartesian_control_msgs::FollowCartesianTrajectoryGoal_<std::allocator<void> >, cartesian_control_msgs::FollowCartesianTrajectoryFeedback_<std::allocator<void> > > {aka class hardware_interface::TrajectoryInterface<cartesian_control_msgs::FollowCartesianTrajectoryGoal_<std::allocator<void> >, cartesian_control_msgs::FollowCartesianTrajectoryFeedback_<std::allocator<void> > >}’ has no member named ‘getHandle’
       cart_traj_interface_.getHandle("cartesian_trajectory_handle").setFeedback(feedback);
                            ^~~~~~~~~
In file included from /home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:28:0:
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/include/ur_robot_driver/hardware_interface.h:244:3: error: ‘cartesian_ros_control’ does not name a type
   cartesian_ros_control::CartesianStateInterface cart_interface_;
   ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/include/ur_robot_driver/hardware_interface.h:245:3: error: ‘cartesian_ros_control’ does not name a type
   cartesian_ros_control::TwistCommandInterface twist_interface_;
   ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/include/ur_robot_driver/hardware_interface.h:246:3: error: ‘cartesian_ros_control’ does not name a type
   cartesian_ros_control::PoseCommandInterface pose_interface_;
   ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/include/ur_robot_driver/hardware_interface.h: In constructor ‘ur_driver::HardwareInterface::HardwareInterface()’:
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/include/ur_robot_driver/hardware_interface.h:282:20: warning: ‘ur_driver::HardwareInterface::cartesian_pose_command_’ will be initialized after [-Wreorder]
   urcl::vector6d_t cartesian_pose_command_;
                    ^~~~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/include/ur_robot_driver/hardware_interface.h:271:20: warning:   ‘urcl::vector6d_t ur_driver::HardwareInterface::joint_positions_’ [-Wreorder]
   urcl::vector6d_t joint_positions_;
                    ^~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:56:1: warning:   when initialized here [-Wreorder]
 HardwareInterface::HardwareInterface()
 ^~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp: In member function ‘virtual bool ur_driver::HardwareInterface::init(ros::NodeHandle&, ros::NodeHandle&)’:
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:336:23: error: ‘JointTrajectoryHandle’ is not a member of ‘hardware_interface’
   hardware_interface::JointTrajectoryHandle joint_trajectory_handle = hardware_interface::JointTrajectoryHandle(
                       ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:336:23: note: suggested alternative: ‘JointTrajectory’
   hardware_interface::JointTrajectoryHandle joint_trajectory_handle = hardware_interface::JointTrajectoryHandle(
                       ^~~~~~~~~~~~~~~~~~~~~
                       JointTrajectory
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:340:23: error: ‘using JointTrajectoryInterface = class hardware_interface::TrajectoryInterface<control_msgs::FollowJointTrajectoryGoal_<std::allocator<void> >, control_msgs::FollowJointTrajectoryFeedback_<std::allocator<void> > > {aka class hardware_interface::TrajectoryInterface<control_msgs::FollowJointTrajectoryGoal_<std::allocator<void> >, control_msgs::FollowJointTrajectoryFeedback_<std::allocator<void> > >}’ has no member named ‘registerHandle’; did you mean ‘registerGoalCallback’?
   jnt_traj_interface_.registerHandle(joint_trajectory_handle);
                       ^~~~~~~~~~~~~~
                       registerGoalCallback
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:340:38: error: ‘joint_trajectory_handle’ was not declared in this scope
   jnt_traj_interface_.registerHandle(joint_trajectory_handle);
                                      ^~~~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:343:23: error: ‘CartesianTrajectoryHandle’ is not a member of ‘hardware_interface’
   hardware_interface::CartesianTrajectoryHandle cartesian_trajectory_handle =
                       ^~~~~~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:343:23: note: suggested alternative: ‘CartesianTrajectory’
   hardware_interface::CartesianTrajectoryHandle cartesian_trajectory_handle =
                       ^~~~~~~~~~~~~~~~~~~~~~~~~
                       CartesianTrajectory
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:348:24: error: ‘using CartesianTrajectoryInterface = class hardware_interface::TrajectoryInterface<cartesian_control_msgs::FollowCartesianTrajectoryGoal_<std::allocator<void> >, cartesian_control_msgs::FollowCartesianTrajectoryFeedback_<std::allocator<void> > > {aka class hardware_interface::TrajectoryInterface<cartesian_control_msgs::FollowCartesianTrajectoryGoal_<std::allocator<void> >, cartesian_control_msgs::FollowCartesianTrajectoryFeedback_<std::allocator<void> > >}’ has no member named ‘registerHandle’; did you mean ‘registerGoalCallback’?
   cart_traj_interface_.registerHandle(cartesian_trajectory_handle);
                        ^~~~~~~~~~~~~~
                        registerGoalCallback
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:348:39: error: ‘cartesian_trajectory_handle’ was not declared in this scope
   cart_traj_interface_.registerHandle(cartesian_trajectory_handle);
                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface_node.cpp:31:0:
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/include/ur_robot_driver/hardware_interface.h:244:3: error: ‘cartesian_ros_control’ does not name a type
   cartesian_ros_control::CartesianStateInterface cart_interface_;
   ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:348:39: note: suggested alternative: ‘cartesian_pose_command_’
   cart_traj_interface_.registerHandle(cartesian_trajectory_handle);
                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~
                                       cartesian_pose_command_
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:350:3: error: ‘cartesian_ros_control’ has not been declared
   cartesian_ros_control::CartesianStateHandle cart_state_handle("base", "tool0_controller", &cart_pose_, &cart_twist_,
   ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:352:3: error: ‘cart_interface_’ was not declared in this scope
   cart_interface_.registerHandle(cart_state_handle);
   ^~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:352:3: note: suggested alternative: ‘js_interface_’
   cart_interface_.registerHandle(cart_state_handle);
   ^~~~~~~~~~~~~~~
   js_interface_
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:352:34: error: ‘cart_state_handle’ was not declared in this scope
   cart_interface_.registerHandle(cart_state_handle);
                                  ^~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/include/ur_robot_driver/hardware_interface.h:245:3: error: ‘cartesian_ros_control’ does not name a type
   cartesian_ros_control::TwistCommandInterface twist_interface_;
   ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:353:3: error: ‘twist_interface_’ was not declared in this scope
   twist_interface_.registerHandle(
   ^~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/include/ur_robot_driver/hardware_interface.h:246:3: error: ‘cartesian_ros_control’ does not name a type
   cartesian_ros_control::PoseCommandInterface pose_interface_;
   ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:353:3: note: suggested alternative: ‘js_interface_’
   twist_interface_.registerHandle(
   ^~~~~~~~~~~~~~~~
   js_interface_
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:354:7: error: ‘cartesian_ros_control’ has not been declared
       cartesian_ros_control::TwistCommandHandle(cart_interface_.getHandle("tool0_controller"), &twist_command_));
       ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:357:3: error: ‘pose_interface_’ was not declared in this scope
   pose_interface_.registerHandle(
   ^~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:357:3: note: suggested alternative: ‘js_interface_’
   pose_interface_.registerHandle(
   ^~~~~~~~~~~~~~~
   js_interface_
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:358:7: error: ‘cartesian_ros_control’ has not been declared
       cartesian_ros_control::PoseCommandHandle(cart_interface_.getHandle("tool0_controller"), &pose_command_));
       ^~~~~~~~~~~~~~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp: In member function ‘virtual void ur_driver::HardwareInterface::read(const ros::Time&, const ros::Duration&)’:
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:562:27: error: ‘using JointTrajectoryInterface = class hardware_interface::TrajectoryInterface<control_msgs::FollowJointTrajectoryGoal_<std::allocator<void> >, control_msgs::FollowJointTrajectoryFeedback_<std::allocator<void> > > {aka class hardware_interface::TrajectoryInterface<control_msgs::FollowJointTrajectoryGoal_<std::allocator<void> >, control_msgs::FollowJointTrajectoryFeedback_<std::allocator<void> > >}’ has no member named ‘getHandle’
       jnt_traj_interface_.getHandle("joint_trajectory_handle").setFeedback(feedback);
                           ^~~~~~~~~
/home/lg/catkin_ws/src/universal_robots/Universal_Robots_ROS_Driver/ur_robot_driver/src/hardware_interface.cpp:604:28: error: ‘using CartesianTrajectoryInterface = class hardware_interface::TrajectoryInterface<cartesian_control_msgs::FollowCartesianTrajectoryGoal_<std::allocator<void> >, cartesian_control_msgs::FollowCartesianTrajectoryFeedback_<std::allocator<void> > > {aka class hardware_interface::TrajectoryInterface<cartesian_control_msgs::FollowCartesianTrajectoryGoal_<std::allocator<void> >, cartesian_control_msgs::FollowCartesianTrajectoryFeedback_<std::allocator<void> > >}’ has no member named ‘getHandle’
       cart_traj_interface_.getHandle("cartesian_trajectory_handle").setFeedback(feedback);
                            ^~~~~~~~~
make[2]: *** [CMakeFiles/ur_robot_driver_plugin.dir/src/hardware_interface.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [CMakeFiles/ur_robot_driver_node.dir/src/hardware_interface_node.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [CMakeFiles/ur_robot_driver_node.dir/src/hardware_interface.cpp.o] Error 1
make[1]: *** [CMakeFiles/ur_robot_driver_plugin.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/ur_robot_driver_node.dir/all] Error 2
make: *** [all] Error 2
cd /home/lg/catkin_ws/build/ur_robot_driver; catkin build --get-env ur_robot_driver | catkin env -si  /usr/bin/make --jobserver-fds=6,7 -j; cd -
.............................................................................................................................................................................
Failed     << ur_robot_driver:make                              [ Exited with code 2 ]

--- a/ur_calibration/README.md
+++ b/ur_calibration/README.md
@@ -53,7 +53,7 @@ respective launchfile in the driver:
 ```bash
 # Replace your actual catkin_ws folder
 $ cd <catkin_ws>/src/example_organization_ur_launch/launch
-$ roscp ur_client_library ur10_bringup.launch ex-ur10-1.launch
+$ roscp ur_robot_driver ur10_bringup.launch ex-ur10-1.launch
 ```
 
 Next, modify the parameter section of the new launchfile to match your actual calibration:

